torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -20,6 +20,7 @@
 import torch.distributed.algorithms.ddp_comm_hooks.post_localSGD_hook as post_localSGD
 import torch.distributed.algorithms.model_averaging.averagers as averagers
 import torch.distributed.algorithms.model_averaging.utils as model_averaging_utils
+from torch.distributed.optim.functional_sgd import _FunctionalSGD
```

Comment:
do we have other functional optimizers supported?

Reply:
Not currently, but @wanchaol has implemented several of these functional optimizers for the RPC framework. It seems pretty feasible to add a `step_param` to all of those and automatically support them (such as adam, rmsprop, etc).